src/handlers/interventions-processor-handler.ts

```diff
@@ -135,12 +143,21 @@ async function handleError(error: unknown, record: SQSRecord) {
     });
   else if (error instanceof StateTransitionError) {
     logger.warn('StateTransitionError caught, message will not be retried.', { errorMessage: error.message });
-    await sendAuditEvent(
-      'AIS_EVENT_TRANSITION_IGNORED',
+    const recordBody = JSON.parse(record.body) as TxMAIngressEvent;
+    await new AuditEvents(
+      TxMAEgressEventTransitionType.TRANSITION_IGNORED,
       error.transition,
-      JSON.parse(record.body) as TxMAIngressEvent,
+      recordBody,
       error.output,
-    );
+    ).send();
+    // await (recordBody.event_name === TriggerEventsEnum.OPERATIONAL_ACCOUNT_INTERVENTION
+    //   ? sendAuditEvent(
+    //       TxMAEgressEventName.AIS_NON_FRAUD_EVENT_TRANSITION_IGNORED,
+    //       error.transition,
+    //       recordBody,
+    //       error.output,
+    //     )
+    //   : sendAuditEvent(TxMAEgressEventName.AIS_EVENT_TRANSITION_IGNORED, error.transition, recordBody, error.output));
   } else {
     logger.error('Error caught, message will be retried.', { errorMessage: (error as Error).message });
```

Comment:
If you update the type for the `error` parameter for this function to this:
```
async function handleError(error: ValidationError | TooManyRecordsError | StateTransitionError | Error, record: SQSRecord) {..}
```
Which makes sense more sense from a Typescript perspective since the error type isn't technically unknown to us. Then you don't need to cast the error type to Error for the `else` block.
```suggestion
    logger.error('Error caught, message will be retried.', { errorMessage: error.message });
```